src/stores/notification.js

```diff
@@ -0,0 +1,91 @@
+import { writable } from 'svelte/store';
+
+function createNotifications() {
```

Comment:
This is example of possible class usage - as we have state here. If you want to continue with function instead - I would probably rename this to something like setupNotificationsService

Reply:
And it is not `service` it is `store`. Don't you mean if I rename it to  `createNotificationStore`

Reply:
Yes, rename to reflect store type please

Reply:
Done